### PR TITLE
[Commercial-AB-Cleanup] Remove appnexus us parameters ab test

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?
Removes appnexus US parameters ab test and brings back the uk-row specific switch

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/header-bidding/utils.spec.js [hunk 1]
```diff
@@ -111,36 +103,55 @@ describe('Utils', () => {
...
         test(`shouldIncludeAppNexus should return true if geolocation is ${region}`, () => {
+            config.switches.prebidAppnexusUkRow = true;
             getSync.mockReturnValue(region);
             expect(shouldIncludeAppNexus()).toBe(true);
         });
     });
 
     ['US', 'CA'].forEach(region => {
-        test(`shouldIncludeAppNexus should return false if geolocation is ${region} and not in variant`, () => {
+        test(`shouldIncludeAppNexus should return false if geolocation is ${region} and`, () => {
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/header-bidding/utils.spec.js:113]:
```suggestion
        test(`shouldIncludeAppNexus should return false if geolocation is ${region}`, () => {
```

File: static/src/javascripts/projects/commercial/modules/header-bidding/utils.spec.js [hunk 2]
```diff
@@ -111,36 +103,55 @@ describe('Utils', () => {
 
     ['AU', 'NZ', 'GB'].forEach(region => {
         test(`shouldIncludeAppNexus should return true if geolocation is ${region}`, () => {
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/header-bidding/utils.spec.js:105]:
```suggestion
        test(`shouldIncludeAppNexus should return true if geolocation is ${region} and the switch is on`, () => {
```

**Review by mxdvl — COMMENTED** [review]
This looks good, but I’m a little unsure about some of the naming.

UK/ROW in this context means: Anywhere in the world except US or CA?

Maybe the testing could be simplified a little bit by using the [test tables][] from jest? I’m thinking something along the lines of:

```js

it.each([
['AU','on', true]
['NZ','on', true]
['GB','on', true]
['US','on', false]
['CA','on', false]
['FK','on', false]
['GI','on', false]
['GG','on', false]
['IM','on', false]
['JE','on', false]
['SH','on', false]
])(`In %s, if switch is %s, shouldIncludeAppNexus should return %s`, (region, switch, expected) => {
  config.switches.prebidAppnexusUkRow = (switch == 'on' ? true : false);
  …
})
```

Otherwise I spotted a few typos in the tests below:

[test tables]: https://jestjs.io/docs/en/api#testeachtablename-fn-timeout

**Comment by ioanna0** [comment]
> Maybe the testing could be simplified a little bit by using the [test tables](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout) from jest? I’m thinking something along the lines of:

@mxdvl That was a great suggestion have changed them into tables & using jest. I have been doing table tests using `forEach` previously but `test.each` from jest is much more precise when a test fails giving you the exact test case that fails in the output ! 💯

**Review by mxdvl — APPROVED** [review]
👍 

> `test.each` from jest is much more precise when a test fails giving you the exact test case that fails in the output ! 💯

That’s great! And they are [reported in TeamCity, too][tc].

[tc]: https://teamcity.gutools.co.uk/viewLog.html?order=DURATION_DESC&recordsPerPage=20&filterText=shouldIncludeAppNexus&status=&currentGroup=test&scope=header-bidding%2futils.spec.js%23teamcity%23*%23teamcity%23*&pager.currentPage=1&buildTypeId=dotcom_master&buildId=570868&tab=testsInfo

**Pull request merged**